setup.py

```diff
@@ -137,9 +137,9 @@ def check_cuda_torch_binary_vs_bare_metal(cuda_dir):
     version_ge_1_5 = ['-DVERSION_GE_1_5']
 version_dependent_macros = version_ge_1_1 + version_ge_1_3 + version_ge_1_5
 
-if "--distributed_adam" in sys.argv:
+if "--distributed_adam" in sys.argv or "--cuda_ext" in sys.argv:
     from torch.utils.cpp_extension import CUDAExtension
-    sys.argv.remove("--distributed_adam")
+    #sys.argv.remove("--distributed_adam")
```

Comment:
These lines are commented so that the build doesn't require the extension-specific flag to be specified.